packages/sdks/src/constants/extra-components.ts

```diff
@@ -33,6 +35,7 @@ export const getExtraComponents: () => RegisteredComponent[] = () => [
         { component: FormInput, ...formInputComponentInfo },
         { component: FormSubmitButton, ...formSubmitButtonComponentInfo },
         { component: FormSelect, ...formSelectComponentInfo },
+        { component: Tabs, ...tabsComponentInfo },
```

Comment:
unfortunately `extra-components` is specifically to skip those components from React Native.

So by moving Tabs here it no longer gets included in RN SDK.

I think we should keep it in the original `builder-registered-components.ts` and add a `TARGET === 'rsc'` check there.